geometry_msgs/PoseStamped pose
---
bool success
string action_id
